vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/main/java/com/vaadin/flow/component/datepicker/DatePicker.java

```diff
@@ -257,6 +264,7 @@ private DatePicker(LocalDate initialDate, boolean isInitialValueOptional) {
         addValueChangeListener(e -> validate());
 
         getElement().addEventListener("unparsable-change", event -> {
```

Comment:
I don't think I was familiar with this concept yet. `unparsable-changed` is fired every time the user tries to commit a value, regardless whether the previous value was already unparsable? The event name made it sound like it's only fired when the component state switches from parsable to unparsable and vice versa.

Reply:
Note that the event name is `unparsable-change` rather than `unparsable-changed`. The event compliments `change` and, when used together, they allow you to get notified of any input changes, including unparsable ones, as if it was a regular text field.

Reply:
OK, thanks